schemas/2015-11-01/Microsoft.Network.json

```diff
@@ -0,0 +1,231 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-11-01/Microsoft.Network.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Network",
+  "description": "Microsoft Network Resource Types",
+  "resourceDefinitions": {
+    "trafficManagerProfiles": {
+      "type": "object",
+      "description" : "An Azure Traffic Manager profile",
+      "properties": {
+        "apiVersion": {
+          "enum": [
+            "2015-11-01"
+          ]
+        },
+        "type": {
+          "enum": [
+            "Microsoft.Network/trafficManagerProfiles"
+          ]
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "profileStatus": {
+              "anyOf": [
+                {
+                  "enum": [
+                    "Enabled",
+                    "Disabled"
+                  ]
+                },
+                {
+                  "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
```

Comment:
Allowing an expression in an "anyOf" clause basically bypasses the constraint on the enum (or integer elsewhere) - anything that doesn't is not "Enabled" or "Disabled" will fall under the "expression" and validate fine. Is there any way to specify that the result of the expression must respect the constraint of the enum (or integer elsewhere)?